### DISCUSSION TITLE
v1.8.1 and LG Washer Dryer

### DISCUSSION THREAD

**Discussion by CyberGiant443** [nVuln/homebridge-lg-thinq#282] [General]
Anyone else having issues with the dryer disappearing when you upgrade to either 1.8.0 or 1.8.1 from 1.7.0? I can recreate the problem by upgrading and fixing it by downgrading back to 1.7.0. Seems like the newer versions remove the tiles from displaying next to the faucet for the washer and remove both the faucet and tile for the dryer. If the dryer is operating the faucet will appear at the top, but the larger faucet that should always be their never shows up. 

Below is version 1.7.0 with everything working. 
![IMG_6094](https://github.com/nVuln/homebridge-lg-thinq/assets/121060157/1f9187b6-730c-4d9f-aee1-3c1c6d5e658d)

**Comment by nVuln** [comment]
you mean you want remove faucet tile?

**Comment by CyberGiant443** [comment]
No. I’m aware that the faucets have to stay because that’s the only way it can work right now in HomeKit. My issue is that when I upgrade from 1.7.0 to either 1.8.0 of 1.8.1 I lose the faucet tile for the Dryer and both the tiles for the Washer and Dryer. This leaves me with just the faucet for the washer.  When I downgrade to 1.7.0 everything returns.

**Reply by nVuln** [reply]
please try `Remove All Cached Accessories`  and `Reset Homebridge Accessory` in Homebridge, let them refresh
I make some change in washer, to fix my mistake in prevous version

**Reply by CyberGiant443** [reply]
I will do so and let you know if it fixes the issue. Does 1.8.1 still show the extra tiles or are they supposed to be gone now?

**Reply by CyberGiant443** [reply]
I followed your instructions and it’s fixed. Thank you for getting back to me so quickly. Here is the updated screenshot.
![IMG_6098](https://github.com/nVuln/homebridge-lg-thinq/assets/121060157/6c5eb0e6-efc8-45f4-bba5-8d6ee02cdb96)

**Reply by nVuln** [reply]
in version 1.8, no more useless tile, if you still want that tub clean event, you can enable it in plugin setting

**Reply by CyberGiant443** [reply]
How do you enable the tub clean event? I checked the configuration and I didn’t see any check box for it. On version 1.8.1 now.

**Reply by nVuln** [reply]
<img width="738" alt="Screenshot 2024-03-04 at 3 32 09 PM" src="https://github.com/nVuln/homebridge-lg-thinq/assets/54855446/016d1bd3-a136-48d9-8d09-228e6d0562a8">

**Comment by CyberGiant443** [comment]
My configuration doesn’t look like yours. I’m on v. 1.8.1. 
![IMG_6124](https://github.com/nVuln/homebridge-lg-thinq/assets/121060157/cd2f2c89-9cf0-4254-aaef-6bcec4f6f28a)

**Reply by nVuln** [reply]
you're using Hoobs? never tested it on Hoobs
you can manual modify a json config like this
```json
        {
            "id": "911fad63-e876-1e0c-b895-xxxxxxx",
            "name": "Washer",
            "type": "WASHER",
            "washer_door_lock": false,
            "washer_tub_clean": true,
            "washer_trigger": true
        }
```

**Reply by CyberGiant443** [reply]
Yes. I’m on Hoobs v4.31. I will attempt to add the additional line to the JSON. Thanks.

**Reply by CyberGiant443** [reply]
I modified the json and now my occupancy sensor always shows detected. Do you know why? What exactly is the washer clean tub feature? Should I have that and the washer door lock set to true also? 
![IMG_6143](https://github.com/nVuln/homebridge-lg-thinq/assets/121060157/4f58e326-f446-4a88-ac4d-3fe2f2c996ca)

**Reply by nVuln** [reply]
that occupancy sensor will be triggered when washer need to run tub clean

**Reply by CyberGiant443** [reply]
I guess it’s time to run the tub clean cycle. 😂 Thanks for all your help. Great plugin.

**Reply by CyberGiant443** [reply]
Is there any reason that the washer door lock always shows as my washer is unlocked. On dryer it has no effect. Should I just disable it in the JSON or is there a way to get it to work for washer and or dryer? Thanks again

**Reply by nVuln** [reply]
disable if it's not worked

**Comment by cheetahbridge3** [comment]
@nVuln I have both a Washer and a Mini Washer. They each have their own Device ID's that I can see in the Homebridge UI. But I'm not seeing the Mini Washer in HomeKit. I only see the Washer. Any thoughts why that might be happening?

**Reply by nVuln** [reply]
hmm, not sure what happened, please turn on homebridge debug mode and collect some debug logging
then create new issue for easy tracking, I'll investigate it
